### ISSUE TITLE
Check and warn (or correct?) heading numbering when one level is skipped

### ISSUE THREAD

**Issue by costateixeira** [HL7/ig-template-base#139]
When a xhtml or markdown page skips one heading level (e.g. start with a h4 / #### without any h3 / ###) the numbering gets confused.

Perhaps we could do a check or a warning (or a refactoring of the numbering mechanism). 

https://chat.fhir.org/#narrow/stream/179252-IG-creation/topic/Duplicate.20header.20numbers.3F

**Comment by Healthedata1** [comment]
if we go away from css based solutiion then issue may disappear

**Comment by costateixeira** [comment]
Indeed, I think we have sufficient arguments to consider that change
